src/snowcli/cli/project/definition_manager.py

```diff
@@ -29,59 +29,75 @@ def __init__(self, project_arg: Optional[str] = None) -> None:
         search_path = os.getcwd()
         if project_arg:
             search_path = os.path.abspath(project_arg)
-        config_files = self._find_definition_files(Path(search_path))
-        if not config_files:
+        project_root = DefinitionManager.find_project_root(Path(search_path))
+        if not project_root:
             raise MissingConfiguration(
                 f"Cannot find project definition (snowflake.yml). Please provide a path to the project or run this command in a valid project directory."
             )
-        self._project_config_paths = config_files
 
-    def _find_definition_files(self, project_path: Path) -> Optional[List[Path]]:
+        self.project_root = project_root
+        self._project_config_paths = self._find_definition_files(self.project_root)
+
+    @staticmethod
+    def _find_definition_files(project_root: Path) -> List[Path]:
```

Comment:
Do static methods should start with `_`? 🤔

Reply:
My understanding was that a single leading underscore simply denotes that the member is not part of the public API for the class. That's what's intended here. If there is a more idiomatic way to achieve this, please let me know and I'll update.